test-interface/src/main/scala/org/scalajs/testinterface/HTMLRunner.scala

```diff
@@ -379,6 +396,7 @@ protected[testinterface] object HTMLRunner extends js.JSApp {
       var className: String = js.native
       val style: Style = js.native
       var onclick: js.Function0[Boolean] = js.native
+      def insertAdjacentElement(location: String, element: Element): Unit = js.native
```

Comment:
According to the [browser compat table of MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentHTML), I hope so.